src/ucp/wireup/select.c

```diff
@@ -1648,25 +1669,24 @@ ucp_wireup_add_bw_lanes(const ucp_wireup_select_params_t *select_params,
             remote_num_paths++;
 
             /* Remove that skipped path from score computation */
-            dev_count.local_skip[dev_index]      = 1;
-            dev_count.remote_skip[ae->dev_index] = 1;
+            dev_count->local_skip[dev_index]      = 1;
+            dev_count->remote_skip[ae->dev_index] = 1;
         }
 
-        if (dev_count.local[dev_index] >= local_num_paths) {
+        if (dev_count->local[dev_index] >= local_num_paths) {
```

Comment:
we also need to make sure that local_num_paths/remote_num_paths also get +1 line 1668/1669. this can be done by also moving skip_dev_index at same level as dev_count, maybe there is simpler way like checking local_skip/remote_skip